### PR TITLE
Add New EPSM and BM Telemetry

### PR DESCRIPTION
**Pull request by mlang25**
This PR adds telemetry points that will allow mission control to better monitor the EPSM and BMs 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have read and followed the `CONTRIBUTING.md` file
- [ ] I have applied the .clang-format file to my changes
- [ ] I have commented my code using Doxygen style comments (see Contributing MD for examples), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the design documentation 
- [ ] I have updated any relevant parameters in the internal ICD
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If this is a bug-fix, I have added tests that can catch similar bugs in the future
- [ ] New and existing unit tests pass locally with my changes
- [ ] For cfe apps I have reserved an empty address for my msgs_ids and perf_ids from the [tracking list](https://docs.google.com/spreadsheets/d/1yRUhNTRBOiGLswlsWftNaYlwGKDeNQ_fCnDYFrc_7fU/edit#gid=772812381) and checked that it does not clash with other apps
- If any message is changed, removed, or added:
  - [ ] Please contact `#software` on Slack about the change(s)
  - [ ] **ONLY IF TELEMETRY MESSAGE** (for now): `./message-utilities/src/utilities.c` is updated with the correct field(s) properly marshalled to JSON **(IMPORTANT)**
  - If a new struct/message is added:
    - [ ] There is a timestamp included via either an explicit timestamp variable in the struct or buried in the a tlmHeader
  - If any of the existing struct's properties change in any way (property name change, property moves around in the struct, property removed):
    - [ ] **Breaking change** is checkmarked above as any change will affect every software downstream

### CODE REVIEW

**Review by byte125 — APPROVED** [review]

File: fsw/public_inc/epsm_msgs.h [hunk 1]
```diff
@@ -86,19 +86,39 @@ static_assert(((CONVERTER_STATE_LEN % 2) == 0),
...
+    uint8_t valid;            // if the msg is valid;
+    uint8_t __padding;
+} TemperatureData_t;
+
+#define TEMPERATURE_DATA_LEN sizeof(TemperatureData_t)
+#if (__STDC_VERSION__ >= 201112L)   // check if compiling with C11 or newer
+// Preprocessor struct size and alignment checks
+static_assert((16 == TEMPERATURE_DATA_LEN),
+              "TemperatureData_t struct size incorrect (expected 16 bytes)");
+static_assert(((BATTERY_MODULE_STATE_LEN % 2) == 0),
```
Comment (srini-cmu) [inline fsw/public_inc/epsm_msgs.h:118]: Shouldn't this be `static_assert(((TEMPERATURE_DATA_LEN% 2) == 0)`?

**Review by srini-cmu — DISMISSED** [review]
Please address the comment I added. Rest of it looks good.

File: fsw/public_inc/epsm_msgs.h (hunk 1 above)
Comment (mlang25) [reply fsw/public_inc/epsm_msgs.h:118]: Yes you are correct. I will make the change

**Comment by mlang25** [comment]
@srini-cmu Can you please just reapprove the PR? I think the update requires a new approval

**Review by epicstar — APPROVED** [review]
I have approved.

**Pull request merged**